data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,12 @@
+/* exported countValues */
+
+function countValues(stack) {
+  const array = [];
+  if (stack.peek() !== undefined) {
+    while (stack.peek() !== undefined) {
+      array.push(stack.pop());
+    }
+    return array.length;
+  }
```

Comment:
That is a way to do it, but using a counter variable would be a bit more efficient.